src/studio/src/designer/backend/Services/Implementation/LanguagesService.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+using Altinn.Studio.Designer.Services.Interfaces;
+
+namespace Altinn.Studio.Designer.Services.Implementation
+{
+    /// <summary>
+    /// Relevant text resource functions
+    /// </summary>
+    public class LanguagesService : ILanguagesService
+    {
+        private readonly IAltinnGitRepositoryFactory _altinnGitRepositoryFactory;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="altinnGitRepositoryFactory">IAltinnGitRepository</param>
+        public LanguagesService(IAltinnGitRepositoryFactory altinnGitRepositoryFactory)
+        {
+            _altinnGitRepositoryFactory = altinnGitRepositoryFactory;
+        }
+
+        /// <summary>
+        /// Returns the languages when the text-files appear in new format
+        /// </summary>
+        /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+        /// <param name="repo">Repository identifier which is unique within an organisation.</param>
+        /// <param name="developer">Username of developer currently working in the repo.</param>
+        /// <returns>The languages</returns>
```

Comment:
Consider using the [inheritdoc](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc) comment here so we dont need to duplicate the documentation from the interface.
Example: https://github.com/Altinn/app-lib-dotnet/blob/chore/90-appbase-simplified/src/Altinn.App.Core/Implementation/NullTaskProcessor.cs